sky/engine/platform/fonts/apple/FontCacheIOS.mm

```diff
@@ -94,15 +94,22 @@
         CTFontCreateWithFontDescriptor(desc.get(), ui_desc.pointSize, nullptr)};
   }
 
-  CFRef<CTFontDescriptorRef> fallbackFont(unichar glyph) {
+  CFRef<CTFontDescriptorRef> fallbackFont(UChar32 glyph) {
```

Comment:
Do we not have the right warnings turned on for this or the previous code to cause a warning?